packages/lit-html/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 3.3.0
+
+### Minor Changes
+
+- [#4901](https://github.com/lit/lit/pull/4901) [`c9160405`](https://github.com/lit/lit/commit/c9160405deaf8de68bb1e587ef9b2484cb58b353) Thanks [@maxpatiiuk](https://github.com/maxpatiiuk)! - Make disabling dev mode warnings simpler
```

Comment:
Is this really deserving of a minor version bump?

Reply:
an argument could be made that it's technically a new feature, and we are doing a minor bump to the `lit` package since reactive-element and lit-element are getting one with the new option, so seems okay?